File: FastForward Planner/src/edu/uno/ai/planning/ff/FastForwardPlanner.java
package edu.uno.ai.planning.ff;

import edu.uno.ai.planning.Planner;
import edu.uno.ai.planning.Problem;
import edu.uno.ai.planning.ss.StateSpaceProblem;

public class FastForwardPlanner extends Planner<FastForwardSearch> {

	public FastForwardPlanner() {
		super("FF");
	}

	@Override
	protected FastForwardSearch makeSearch(Problem problem) {
		return new FastForwardSearch(new StateSpaceProblem(problem));
	}

}

File: UnitySokoban/Assets/Scripts/Planning/FastForward/FastForwardPlanner.cs
using Planning;
using StateSpaceSearchProject;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace FastForward
{
    public class FastForwardPlanner : Planner<FastForwardSearch> {

        public FastForwardPlanner()
            :base("FF")
        {
        }

        public override FastForwardSearch makeSearch(Problem problem)
        {
            return new FastForwardSearch(new StateSpaceProblem(problem));
        }
    }

}
